src/Microsoft.AspNetCore.Mvc.Formatters.Json/Internal/MvcJsonMvcOptionsSetup.cs

```diff
@@ -29,38 +35,40 @@ public class MvcJsonMvcOptionsSetup : ConfigureOptions<MvcOptions>
             IOptions<MvcJsonOptions> jsonOptions,
             ArrayPool<char> charPool,
             ObjectPoolProvider objectPoolProvider)
-            : base((options) => ConfigureMvc(
```

Comment:
Feel free to remove the doc comments for this ctor. The params are incomplete anyway.